File: spring-web/src/main/java/com/example/springwebex/SpringExApplication.java
package com.example.springwebex;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.mybatis.spring.annotation.MapperScan;
import lombok.extern.slf4j.Slf4j;


@SpringBootApplication
@Slf4j
public class SpringExApplication {

    public static void main(String[] args) {
        SpringApplication.run(SpringExApplication.class, args);
        log.info("***** main() *****");
    }
}

File: spring-web-kotlin/src/main/kotlin/com/example/springwebex/SpringExApplication.kt
package com.example.springwebex

import io.github.oshai.kotlinlogging.KotlinLogging
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class SpringExApplication

private val log = KotlinLogging.logger {}

fun main(args: Array<String>) {
    runApplication<SpringExApplication>(*args)
    log.info { "***** main() *****" }
}
